### PR TITLE
Fix TinyMCE images directory options if '/administrator/images' exists

### PR DESCRIPTION
**Pull request by korenevskiy**
By creating the IMAGES directories in the ADMINISTRATOR directory. 
Create the TEST subdirectory in the IMAGES directory. 
Go to the settings of the Editor Tinymce plugin and select a subfolder to save images. 
We see that the list for selecting subfolders is limited to only one TEST folder. 
Thus, the list of the directory from the administrator folder is displayed in the plugin settings. But when saving image files in the materials manager, the files are still saved to the standard image folder. The presence of the IMAGES folder in the Admin folder breaks the configuration setting of the Tinymce editor plugin.

Fix bug fo ISSUE https://github.com/joomla/joomla-cms/issues/39946

Addition: 
# You need to test with having previously created the IMAGES directory in the ADMINISTRATOR directory. Otherwise, there is no point in testing.

### Addition @brianteeman

For others looking at this. It is an obscure but valid bug.

The images directory option in TinyMCE should look something like this

Image

In other words it should be displaying the /images directory and all subdirectories.

The bug occurs if you also have a folder /administrator/images

That should have no effect on the field in tiinymce but as shown below it does
Image
image

### CODE REVIEW

**Comment by joomdonation** [comment]
I have tested this item :white_check_mark: successfully on 8d4093c4e6c26054faf42ed4aa5c49365b78a0bd<br /><br />Works as expected. Thanks!<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/39962">issues.joomla.org/tracker/joomla-cms/39962</a>.</sub>

**Comment by joomdonation** [comment]
For testers, please see this comment https://github.com/joomla/joomla-cms/issues/39946#issuecomment-1446983018 from @brianteeman . It explains the issue this PR solves clearer than the description of this PR.

**Comment by viocassel** [comment]
I have tested this item :white_check_mark: successfully on 8d4093c4e6c26054faf42ed4aa5c49365b78a0bd<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/39962">issues.joomla.org/tracker/joomla-cms/39962</a>.</sub>

**Comment by korenevskiy** [comment]
> I have tested this item ✅ successfully on [8d4093c](https://github.com/joomla/joomla-cms/commit/8d4093c4e6c26054faf42ed4aa5c49365b78a0bd)
> 
> This comment was created with the [J!Tracker Application](https://github.com/joomla/jissues) at [issues.joomla.org/tracker/joomla-cms/39962](https://issues.joomla.org/tracker/joomla-cms/39962).

Have you tested with the IMAGES directory in the ADMINISTRATOR directory or not?

**Comment by joomdonation** [comment]
> Have you tested with the IMAGES directory in the ADMINISTRATOR directory or not?

Yes. I tested.

**Comment by joomdonation** [comment]
RTC. Thanks.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/39962">issues.joomla.org/tracker/joomla-cms/39962</a>.</sub>

**Comment by fancyFranci** [comment]
Great that you fixed this really obscure bug! Thank you :)
